Refactor mongoBenchmarker to use grep to pipe out execution stats

ecuyle opened the pull request:
@code-mel 
Please review the following PR:
mongoBenchmarker.sh and pgBenchmarker.sh are automated testing scripts that perform benchmarking logic for me. Please review both scripts. Thanks!

ecuyle committed:
Refactor mongoBenchmarker to use grep to pipe out execution stats

code-mel commented on database/mongoBenchmarker.sh:
```diff
@@ -18,8 +18,8 @@ echo -e "Starting benchmarking at: $start\n"
 # Reads
-runMongoTest reservations "db.reservations.find({ id: 9999999 })"
-runMongoTest restaurants "db.restaurants.find({ id: 9999999 })"
+runMongoTest reservations "db.reservations.find({ id: 9999999 }).explain('executionStats')"
```
This is cool, just looked into it .explain()

ecuyle committed:
Add PG connection file and cleanup DB dir

ecuyle committed:
Implement backend API routes

Implements GET to /api/reserve/load/:id and
/api/reserve/query/:id/:date/:time. Implements POST to
/api/reserve/query/:id/:date/:time. Front end connects smoothly now with
backend.
